dl/src/main/scala/com/intel/analytics/bigdl/nn/Cosine.scala

```diff
@@ -182,6 +183,10 @@ class Cosine[T: ClassTag](val inputSize : Int, val outputSize : Int)(
     (Array(this.weight), Array(this.gradWeight))
   }
 
+  override def getParametersTable(): Table = {
+    T(getName() -> T("weight" -> weight, "gradWeight" -> gradWeight))
```

Comment:
@yiheng @qiuxin2012  maybe we can some how move weight or bias to an abstract class, then eliminate those duplicate *parameters() code in the following PR. 
What it want to get is simple, if bias is not null, add bias and if weight is not null add weight, but currently they are spread across many layers.